Makefile

```diff
@@ -1,4 +1,3 @@
-DC := $(shell which docker-compose)
```

Comment:
On second thoughts, if we want to support `pacman` (and `pacman-compose`), we might be better off keeping DC?

Also, is `command -v` more portable/reliable than `which`?

Untested

```makefile
DOCKER := $(shell command -v docker || command -v pacman || (echo "Error: neither docker nor pacman found" >&2 && exit 1))

DC := $(shell \
	if command -v docker > /dev/null && docker compose version > /dev/null 2>&1; then \
		echo "$$(command -v docker) compose"; \
	elif command -v pacman-compose > /dev/null; then \
		echo "$$(command -v pacman-compose)"; \
	elif command -v docker-compose > /dev/null; then \
		echo "$$(command -v docker-compose)"; \
	else \
		echo "Error: no suitable docker compose command found" >&2; \
		exit 1; \
	fi \
)
```

Reply:
I'd keep these changes outside this PR, which only ensures compatibility with the recent Docker versions.

Also, Jiri made padman to work with docker commands by using symlinks.